### PR TITLE
Feat: Lido V2 initial docs rework [batch 1]

### PR DESCRIPTION
**Pull request by TheDZhon**
- [x] Rename contracts
- [x] Update sidebar
- [x] Remove Terra mentions :new_moon_with_face: 
- [x] Remove Rinkeby, Kovan, and Ropsten deployments (testnets are discontinued)
- [x] Remove stETH/ETH deprecated price oracle mentions 
- [x] Stubs for the new and outdated contracts
- [x] Remove 'withdrawals aren't available yet claims' and include withdrawals support feature here and there as a part of the protocol
- [x] Some incomplete edits and attempts to catch up on the current protocol state :disappointed:

### CODE REVIEW

File: docs/contracts/validators-exit-bus-oracle.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# ValidatorsExitBusOracle
+
+- [Source code](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/oracle/ValidatorsExitBusOracle.sol)
+- [Deployed contract](https://etherscan.io/address/0x0De4Ea0184c2ad0BacA7183356Aea5B8d5Bf5c6e)
+
+A contract implementing a message bus between the on-chain protocol and off-chain observers
```
Comment (folkyatina) [inline docs/contracts/validators-exit-bus-oracle.md:6]: Between oracle and node operators actually. Both are off-chain.

File: docs/guides/protocol-levers.md [hunk 2]
```diff
@@ -11,125 +11,121 @@ All existing levers are listed below, grouped by the contract.
...
+- [`AccountingOracle`](/contracts/accounting-oracle)
+- [`ValidatorsExitBusOracle`](/contracts/validators-exit-bus-oracle)
+- [`WithdrawalVault`](/contracts/withdrawal-vault)
+- [`WithdrawalQueueERC721`](/contracts/withdrawal-queue-erc721)
+- [`LegacyOracle`](/contracts/legacy-oracle)
 
-Upgradeability is implemented by the Aragon kernel and base contracts. To upgrade an app, one needs
-the `dao.APP_MANAGER_ROLE` permission provided by Aragon. All upgradeable contracts use the
-[Unstructured Storage pattern] in order to provide stable storage structure across upgrades.
+Upgradeability is implemented either by the Aragon kernel and base contracts OR by the [OssifiableProxy](/contracts/ossifiable-proxy) instances.
```
Comment (folkyatina) [inline docs/guides/protocol-levers.md:24]: WithdrawalVault is not behind the OssifiableProxy AFAIK

File: docs/token-guides/steth-superuser-functions.md [hunk 3]
```diff
@@ -7,24 +7,28 @@
 StETH token is the upgradable contract behind `AppProxyUpgradeable` proxy at [https://etherscan.io/address/0xae7ab96520de3a18e5e111b5eaab095312d7fe84](https://etherscan.io/address/0xae7ab96520de3a18e5e111b5eaab095312d7fe84). Lido DAO can change the implementation with the successful DAO vote.
 
 StETH can be stopped by the DAO vote. No operations changing stETH balances can be performed on the stopped contract:
+
 1. `approve` and `transfer` calls revert;
```
Comment (folkyatina) [inline docs/token-guides/steth-superuser-functions.md:11]: approve is possible.

**Review by folkyatina — APPROVED** [review]
Left several comments, but it's much better than it was. Thank you.

File: docs/contracts/validators-exit-bus-oracle.md (hunk 1 above)
Comment (TheDZhon) [reply docs/contracts/validators-exit-bus-oracle.md]: Thank you, agree :+1:

File: docs/guides/protocol-levers.md (hunk 2 above)
Comment (TheDZhon) [reply docs/guides/protocol-levers.md:24]: It's also an ossifiable proxy, though, the source code is different, it's correct

Comment (TheDZhon) [reply docs/guides/protocol-levers.md:24]: Pushed a more accurate version, thx!

File: docs/token-guides/steth-superuser-functions.md (hunk 3 above)
Comment (TheDZhon) [reply docs/token-guides/steth-superuser-functions.md:11]: oh, nice catch!

**Pull request merged**